rust/fastsim-cli/src/bin/fastsim-cli.rs

```diff
@@ -290,7 +290,7 @@ pub fn main() {
             traceMissInMph: sdl.0.trace_miss_speed_mph,
             h2AndDiesel: None,
         };
-        println!("{}", res.to_json());
+        println!("{}", res.to_json().unwrap());
```

Comment:
@kylecarow, I'm not certain about this, but maybe it'd be better to have the containing function return a result and use `?` rather than `.unwrap()`

Reply:
I agree that this function should return a Result, just put these unwraps to make the code compile (and there are other unwraps in this function already).

I'll change the function to return `anyhow::Result` and change these unwraps to `?`s as appropriate, as that would fit well in this PR

Reply:
done!